rfml.go

```diff
@@ -446,7 +482,25 @@ func uploadRFMLFilesInDirectory(rfmlDirectory string) error {
 	// And here we update all of the tests
 	testsToUpdate := make(chan *rainforest.RFTest, len(parsedTests))
 	for _, testToUpdate := range parsedTests {
-		testToUpdate.PrepareToUploadFromRFML(mappings)
+		testID, ok := mappings.MapRFMLIDtoID()[testToUpdate.RFMLID]
+		if ok {
+			testToUpdate.TestID = testID
```

Comment:
Only if it exists. I had to change the logic around so that I always had the test ID before uploading embedded files.